partcad/src/partcad/runtime_python.py

```diff
@@ -110,20 +110,13 @@ async def once_async(self):
             async with self.get_async_lock():
                 if not self.initialized:
                     # Preinstall the most common packages to avoid
-                    # TODO(clairbee): Lock the entire runtime instead
-                    await self.ensure_async_onced_locked(
-                        "ocp-tessellate==3.0.8"
-                    )
+                    await self.ensure_async_onced_locked("ocp-tessellate==3.0.8")
                     await self.ensure_async_onced_locked("cadquery==2.4.0")
                     await self.ensure_async_onced_locked("numpy==1.26.4")
-                    await self.ensure_async_onced_locked(
-                        "numpy-quaternion==2023.0.4"
-                    )
+                    await self.ensure_async_onced_locked("numpy-quaternion==2023.0.4")
                     await self.ensure_async_onced_locked("nptyping==2.0.1")
                     # await self.ensure_async_onced_locked("typing_extensions>=4.6.0,<5") # doesn't work on Windows
-                    await self.ensure_async_onced_locked(
-                        "typing_extensions==4.12.2"
-                    )
+                    await self.ensure_async_onced_locked("typing_extensions==4.12.2")
```

Comment:
I do feel like those should be in some `requirements.txt` but I don't know this code to be sure.

Reply:
I agree